var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -45,6 +45,10 @@ class Adios2(CMakePackage, CudaPackage, ROCmPackage):
     version("2.4.0", sha256="50ecea04b1e41c88835b4b3fd4e7bf0a0a2a3129855c9cc4ba6cf6a1575106e2")
     version("2.3.1", sha256="3bf81ccc20a7f2715935349336a76ba4c8402355e1dc3848fcd6f4c3c5931893")
 
+    depends_on("c", type="build")  # generated
+    depends_on("cxx", type="build")  # generated
+    depends_on("fortran", type="build")  # generated
```

Comment:
We might want to update the generation script to look for `variant("lang")` in the packages. `adios2` only depends on `fortran` when building the fortran bindings with `+fortran`.

Reply:
I'm ok with doing this in a follow-on PR to get the initial state in.  I do think this is a good idea.